data-products-dp0-2/index.rst

```diff
@@ -168,42 +168,53 @@ Note that in the future, for real LSST data releases, this level of redundancy i
 
 
 .. list-table:: Catalog data available for DP0.2.
-   :widths: 100 100 390
+   :widths: 100 100 390 100
    :header-rows: 1
 
    * - TAP Name
      - Butler Name
      - Description
+     - Link to Table
    * - Object
      - objectTable
      - Astrometric and photometric measurements for objects detected in coadded images (990 columns).
+     - `Object <https://data.lsst.cloud/portal/app/?api=tap&service=https://data.lsst.cloud/api/tap&schema=dp02_dc2_catalogs&table=dp02_dc2_catalogs.Object>`_
    * - Source
      - sourceTable
      - Astrometric and photometric measurements for sources detected in the individual PVIs (143 columns).
+     - `Source <https://data.lsst.cloud/portal/app/?api=tap&service=https://data.lsst.cloud/api/tap&schema=dp02_dc2_catalogs&table=dp02_dc2_catalogs.Source>`_
    * - ForcedSource
      - forcedSourceTable
      - Forced photometry on the individual PVIs at the locations of all detected objects (38 columns).
+     - `ForcedSource <https://data.lsst.cloud/portal/app/?api=tap&service=https://data.lsst.cloud/api/tap&schema=dp02_dc2_catalogs&table=dp02_dc2_catalogs.ForcedSource>`_
```

Comment:
This name and others from this "Name" column overlap with the next column, when rendering. Could it be made bigger?: 

<img width="869" alt="image" src="https://github.com/user-attachments/assets/6e5f8d44-a1f2-49ba-8695-2075738f8fe5">

Reply:
hmmm my view of the rendered page is ok, but maybe the table is too long to be compatible with various types of displaying environments. So I removed the 'Link to Table' column and put the links under the 'TAP Name' columns.

Reply:
That helped with some rows, but I still see one row that overlaps: 

<img width="827" alt="image" src="https://github.com/user-attachments/assets/0454ac1a-4662-442d-b02c-02ab5187b25b">

It might be browser-dependent then. The above screenshot is with Chrome. But the one below, with Safari, and it looks fine as you say: 

<img width="743" alt="image" src="https://github.com/user-attachments/assets/94246fdf-42c3-4f86-9f19-b67459253f46">